data/news/san_mateo.json

```diff
@@ -59,13 +66,6 @@
       "date_published": "2020-09-04T22:08:15Z",
       "summary": "Additional Evacuees May Return Home --\u00a0Non-Residents Urged to Stay Out of Area"
     },
-    {
-      "id": "12486 at https://cmo.smcgov.org",
-      "url": "https://cmo.smcgov.org/press-release/sept-3-2020-health-officer%E2%80%99s-holiday-message-county-parks-reopen-update-county",
-      "title": "Sept. 3, 2020 - Health Officer\u2019s Holiday Message; County Parks Reopen; Update on County Response to COVID-19, Testing & Case Count",
```

Comment:
I don't think this should block merging for now, but making a note for myself to investigate this removal and make sure it actually got removed from the source page, since it's certainly within the timeframe and our code should not be stripping it out.

Reply:
As of this afternoon, when I had time to check, the news feed only goes back to September 7th, so it’s hard to say anything meaningful about this now. The next update will wind up removing all the entries around this one.